tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -1380,7 +1380,13 @@ def fit_pseudolabel(self, pseudo_data: pd.DataFrame, max_iter: int = 5, return_p
                                             fit_ensemble=fit_ensemble, fit_ensemble_every_iter=fit_ensemble_every_iter,
                                             **fit_extra_kwargs)
 
-    def predict(self, data, model=None, as_pandas=True, transform_features=True):
+    def predict(self,
+                data,
+                model=None,
+                as_pandas=True,
+                transform_features=True,
+                *,
+                decision_threshold: float = None):
```

Comment:
nit: ditto (types)

Reply:
We can add type hints in this method in a separate PR (Although note that docstring has the types listed)

Reply:
Sounds good;
I would note that types in docstrings are less helpful if we want to add automated types checks via mypy (i.e. like it's done in eda package).

Reply:
added type hints